apps/communications/contacts/js/activities.js

```diff
@@ -233,6 +240,10 @@ var ActivityHandler = {
     }
 
     if (this.activityDataType.indexOf('text/vcard') !== -1) {
+      if (fb.isFbContact(theContact)) {
+        Contacts.confirmDialog(null, 'facebook-export-forbidden', dismiss);
```

Comment:
Was trying it out on the phone, and sounds a bit weird to me:
'The contact cannot be shared due to Facebook constraint.'

Perhaps we need a better wording, not a native speaker, but saying that we have a 'constraint', and not saying which one sounds a bit weird. I would rather say something like 'Facebook contacts cannot be exported'